### PR TITLE
Rel v4.17.0.1

### PR DESCRIPTION
**Pull request by mkopec**
Waiting for test results

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -397,12 +398,85 @@ <h1>Documentation</h1>
...
+                <li>Rebased with official coreboot repository commit 5eda52a</li>
+                <li>updated sortbootorder to v4.6.24</li>
+              </ol>
+              <li>Added:</li>
+              <ol>
+                <li>Support for APU7 (APU3 variant with 2.5GbE i225 NICs)</li>
+              </ol>
+              <li>Known issues:</li>
+              <ol>
+                  <li><a href="https://github.com/pcengines/coreboot/issues/329">apuled driver doesn't work in FreeBSD.</a> Check the <a target="_blank" href="https://github.com/pcengines/apu2-documentation/blob/master/docs/gpios.md">GPIOs</a> document for workaround.</li>
```
Comment (miczyg1) [inline index.html:426]: Shouldn't we create an issue for apu7 lack of iPXE support?

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- Support for APU7 (APU3 variant with 2.5GbE i225 NICs)
+
+### Changed
+
+- **Rebased** with official coreboot repository **commit** 5eda52a
+- **Updated sortbootorder** to **v4.6.24** adding support for APU7
+
+## Statistics
+
+![Files Changed](https://cloud.3mdeb.com/index.php/s/FIXME/preview)
```
Comment (miczyg1) [inline _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md:25]: Missing chart

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+point of given release specified in CHANGELOG (CHANGELOG.md, gitlab-ci.yml
+and regression.sh excluded from statistics). Check the statistics with:
+
+```
+git diff --stat 5eda52a ':(exclude).gitlab-ci.yml' ':(exclude)CHANGELOG.md' ':(exclude).gitlab-ci/regression.sh'
+```
+
+`116 files changed, 4237 insertions(+), 217 deletions(-)`
+
+![Process of mainlining](https://cloud.3mdeb.com/index.php/s/nynFwrwrqLqeSrN/preview)
```
Comment (miczyg1) [inline _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md:37]: Old chart

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+## Testing
+
+* [PC Engines hardware configuration matrix](https://cloud.3mdeb.com/index.php/s/4n9rT4yMsKezHsR/preview) -
+  hardware configurations available for testing in 3mdeb laboratory.
+
+* [PC Engines release validation results](https://docs.google.com/spreadsheets/d/1_uRhVo9eYeZONnelymonYp444zYHT_Q_qmJEJ8_XqJc/edit#gid=0) -
+  please note there are separate sheets for each board-release.
+
+![Mainline test results](https://cloud.3mdeb.com/index.php/s/FIXME/preview)
```
Comment (miczyg1) [inline _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md:50]: Missing chart

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md (hunk 2 above)
Comment (mkopec) [reply _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md:25]: @sulewskiprzemyslaw Could you fill these in as well?

Comment (sulewskiprzemyslaw) [reply _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md]: The link to the chart has been added

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md (hunk 3 above)
Comment (sulewskiprzemyslaw) [reply _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md]: The link to the chart has been improved

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md (hunk 4 above)
Comment (sulewskiprzemyslaw) [reply _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md]: The link to the chart has been added

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md (hunk 2 above)
Comment (mkopec) [reply _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md]: Thanks!

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+* [apu7 v4.17.0.1.zip](https://3mdeb.com/open-source-firmware/pcengines/apu7/apu7_v4.17.0.1.zip)
+
+  [apu7 v4.17.0.1.rom](https://3mdeb.com/open-source-firmware/pcengines/apu7/apu7_v4.17.0.1.rom)
+
+  [SHA256 file](https://3mdeb.com/open-source-firmware/pcengines/apu7/apu7_v4.17.0.1.SHA256)
+
+  [SHA256 sig](https://3mdeb.com/open-source-firmware/pcengines/apu7/apu7_v4.17.0.1.SHA256.sig)
+
+See how to verify the signatures on [asciinema](https://asciinema.org/a/475909)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply _posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md]: Updated

File: index.html [hunk 6]
```diff
@@ -397,12 +398,86 @@ <h1>Documentation</h1>
...
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu2/apu2_v4.17.0.1.zip">apu2 v4.17.0.1.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu3/apu3_v4.17.0.1.zip">apu3 v4.17.0.1.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu4/apu4_v4.17.0.1.zip">apu4 v4.17.0.1.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu5/apu5_v4.17.0.1.zip">apu5 v4.17.0.1.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu6/apu6_v4.17.0.1.zip">apu6 v4.17.0.1.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu7/apu7_v4.17.0.1.zip">apu7 v4.17.0.1.zip</a></li>
+                  </ul>
+                  <p>See an example how to verify the signatures on <a href="https://asciinema.org/a/475909">asciinema</a></p>
```
[the remark this answers is not in the document]
Comment (mkopec) [reply index.html]: Updated

**Pull request merged**